xclim/indicators/land/_snow.py

```diff
@@ -231,3 +237,24 @@ class SnowWithIndexing(ResamplingIndicatorWithIndexing):
     var_name="snd",
     compute=xci.snw_to_snd,
 )
+
+
+snd_days_above = SnowWithIndexing(
+    title="Days with snow (depth)",
+    identifier="snd_days_above",
+    units="days",
+    long_name="Number of days with snow",
```

Comment:
I'm kind of wondering now if we don't want to be clearer with titles like "Days with snow depth greater than threshold"

Reply:
I wondered the same. I think the titles are not always an exact description of what the indicator does, but often a simpler description of the most common use-case (like "growing degree days"). Here, it seems to me the most common usage is to find the number days with a significant amount of snow.